src/modules/mealInfoModal.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable */
+import commentCount from "./commentCounter.js";
+
+const modalDetailsContent = document.querySelector(".modal-details-content");
+const url =
+  "https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/";
+const key = "4ZfW9Nw9KyQJZ8SztYPH";
+
+function mealInfoModal(meal) {
+  const {
+    strMeal,
+    strMealThumb,
+    strCategory,
+    strArea,
+    strIngredient4,
+    strMeasure1,
+    idMeal,
+  } = meal[0];
+  // meal = meal[0];
+  const html = `
+        <div class="modal-meal-img">
+              <img src="${strMealThumb}" alt="" />
+            </div>
+            <h2 class="modal-title">${strMeal}</h2>
+            <div class="item-details">
+              <div>
+                <p>${strCategory}</p>
+                <p>${strArea}</p>
+              </div>
+              <div>
+                <p>${strIngredient4}</p>
+                <p>${strMeasure1}</p>
+              </div>
+            </div>
+            <h2 id="commentTitle">Comments</h2>
+            <div id="commentsDisplay"></div>
+            <h2 id="plusComment">Add a comment</h2>
+            <div id="form">
+            <input type="text" id="username" placeholder="Your name" />
+            <input type="text" id="comment" placeholder="Add Your comment" />
+            <button type="submit" id="submitBtn">Submit</button>
+            </div>
+    `;
+  modalDetailsContent.innerHTML = html;
+  modalDetailsContent.parentElement.classList.add("showModal");
+  const submitBtn = document.getElementById("submitBtn");
+
+  submitBtn.addEventListener("click", () => {
```

Comment:
- [ ] Kindly ensure when the user submits a comment only the popup window is refreshed and not the entire application as is the case currently, this will ensure a better user experience.